go.mod

```diff
@@ -1,8 +1,8 @@
 module code.cloudfoundry.org/cf-operator
 
 require (
-	code.cloudfoundry.org/quarks-job v0.0.0-20191108023631-35b1d105944c
-	code.cloudfoundry.org/quarks-utils v0.0.0-20191107101800-2cb41bebc5ca
+	code.cloudfoundry.org/quarks-job v0.0.0-20191112143024-43eff796dc9b
+	code.cloudfoundry.org/quarks-utils v0.0.0-20191112142553-a2225c40261f
```

Comment:
This is not the same quarks-utils as [quarks-job 43eff796](https://github.com/cloudfoundry-incubator/quarks-job/commit/43eff796dc9bdb6b91dcdca2be240a0875faf325) uses, but I guess that's not a problem.

Reply:
Yeah. It's kind of messed now. Edward's PR should fix these.